song-server/src/main/resources/db/migration/V1_10__create_function_get_analysis.sql

```diff
@@ -24,30 +24,8 @@ returns table (
 ) as
 $$
 	begin
-		 -- step A: analysis_file_join is the result of joining Analysis table with File table, and getting file info from Info table
```

Comment:
you cannot modify existing scripts, you need to add new script since this one is already executed

Reply:
added a new script